### ISSUE TITLE
Implement a snooze button for reviews in the gate column

### ISSUE THREAD

**Issue by jrobbins** [GoogleChrome/chromium-dashboard#2712]
**Describe the solution you'd like**
In today's API Owners meeting, they decided to skip over one of the intents for a while.  They asked for button to snooze the review for a month.   

I think this is a special case of the general "needs work" vote where the reviewers are waiting on the feature owners to follow up, but in this case there was a specific timeframe for the soonest follow-up.

**Additional context**
We have a `next_action` field for each gate.  However, it is not shown in the new UI gate column or in the feature rows yet.

**Comment by jrobbins** [comment]
Merged into issue #2711
